### ISSUE TITLE
Duplicate Items

### ISSUE THREAD

**Issue by AngelaFaulding** [NHSDigital/DataDictionaryPublication#489]
I created an Attribute Terminology for "PERSON STATED EYE COLOUR THE SAME INDICATOR" and I think I must have clicked twice on Create as my machine is slow this morning. There are now two Attribute Terminologies. This seems strange as I would expect that you shouldn't be able to create an item with the same name in a folder.
If we are doing a lot of authoring this could happen. I obviously can delete one of the terminologies but want to check this first.
Should we be prevented form creating the same item twice?

<img width="194" alt="image" src="https://github.com/user-attachments/assets/c3e78510-28a7-4e10-8180-660c267b4d88">

I didn't release this until I tried to create a Term and got an error message:

![image](https://github.com/user-attachments/assets/e18236db-6ca0-48f5-90bf-1dfd88365cbf)

**Comment by pjmonks** [comment]
You are correct, this should not be possible. I would have expected Mauro to validate that another terminology of the same label exists, but due to the timing of both creation requests perhaps this was not possible.

There is a simple fix we can perform on the UI to prevent this from happening which I have created an issue for. It is low priority, but we will get to it:

- MauroDataMapper/mdm-ui#885

For now I would suggest deleting the duplicate and continue adding terms to the remaining terminology.

**Comment by pjmonks** [comment]
This has been fixed and release to the test environment.

**Comment by AngelaFaulding** [comment]
@KatePalmerLilley - we need to test this.

**Comment by KatePalmerLilley** [comment]
@AngelaFaulding @pjmonks  I've tested what is described in https://github.com/MauroDataMapper/mdm-ui/issues/885, and yes the "Submit Terminology" button is disabled as soon as it is clicked, so I believe this particular issue to be fixed. 

However, I was able to create a duplicate attribute terminology if I changed the capitalisation, e.g.:

![image](https://github.com/user-attachments/assets/97628e64-87c7-46df-807e-dab99f73e08f)

Can we discuss.

**Comment by KatePalmerLilley** [comment]
@angela this is ready for testing and I have raised a new ticket for the new duplicate issue described above #595
